packages/manager/modules/hub/src/components/payment-status-tile/payment-status-tile.controller.js

```diff
@@ -0,0 +1,63 @@
+import get from 'lodash/get';
+
+import AutorenewService from './AutorenewService.class';
+import {
+  AUTORENEW_URL,
+  DEFAULT_REGION,
+  NUMBER_OF_RECORDS,
+  SERVICE_STATES,
+} from './payment-status-tile.constants';
+
+export default class PaymentStatusTileCtrl {
+  /* @ngInject */
+  constructor($q, atInternet, coreConfig, OvhApiBillingAutorenewServices) {
+    this.$q = $q;
+    this.atInternet = atInternet;
+    this.coreConfig = coreConfig;
+    this.OvhApiBillingAutorenewServices = OvhApiBillingAutorenewServices;
```

Comment:
you could simply use $http to avoid importing ovh-api-services for just 2 calls

Reply:
Looks like we won't use `ovh-api-services` soon enough with this logic